docs/howto-run-microservices-locally.md

```diff
@@ -33,7 +34,8 @@ The easiest way to start all Microservices is to use the `docker-compose.yml` fi
 
 ## Build and run the Industrial IoT Microservices with Visual Studio or VS Code
 
-1. First, make sure your development tool chain is setup to build the Microservices. If you have not, [install .NET Core 2.1+][dotnet-install] and any recent edition of Visual Studio (Windows/MacOS) or Visual Studio Code (Windows/MacOS/Linux).
+<Todo: need to fill in all relevant urls. Now we should be using vs 2019 and .net core sdk 2.2. If 2019 used, the 2.1 sdk needs to be uninstalled or set the correct MSBuildSdksPath environment variable>
```

Comment:
There is no link here especially .net core 2.2 needs to be used. OPC publisher service depends on .net core 2.2.